### DISCUSSION TITLE
JDK 版本

### DISCUSSION THREAD

**Discussion by LiuStar2233** [XiaoHeiPa/LiquidLunar#39] [Q&A]
这个源码使用的JDK是哪个版本的
![屏幕截图(317)](https://github.com/CubeWhyMC/LiquidLunar/assets/159122356/9933e25d-401e-483d-81af-0b82f6ef6cf2)

**Comment by cubewhy** [comment]
1.8

**Reply by LiuStar2233** [reply]
这是游戏版本，JDK是JDK15吗？

**Comment by cubewhy** [comment]
jdk 1.8
